Observer/Frontend/CustomerAccountEdit.php

```diff
@@ -38,38 +52,38 @@ public function __construct(
         $this->moduleManager = $moduleManager;
         $this->customerModel = $customerModel;
         $this->storeManager = $storeManager;
-        $this->sailthruClient = $clientManager;
+        $this->clientManager = $clientManager;
         $this->sailthruSettings = $sailthruSettings;
         $this->sailthruCookie = $sailthruCookie;
         $this->sailthruCustomer = $sailthruCustomer;
         $this->sailthruVars = $sailthruVars;
     }
 
-    public function execute(Observer $observer) {
+    public function execute(Observer $observer)
+    {
         $websiteId = $this->storeManager->getWebsite()->getWebsiteId();
         $this->customerModel->setWebsiteId($websiteId);
         $email = $observer->getData('email');
         $customer = $this->customerModel->loadByEmail($email);
         $storeId = $customer->getStore()->getId();
-        $this->sailthruClient = $this->sailthruClient->getClient(true, $storeId);
+        $client = $this->clientManager->getClient($storeId);
```

Comment:
I appreciated the code cleanup here (no longer reusing the same member variable for difference purposes). Thanks!